bucket/fq.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.0.4",
+    "description": "Tool, language and decoders for working with binary data.",
+    "homepage": "https://github.com/wader/fq",
+    "license": "MIT",
```

Comment:
Source?

Reply:
for license?

Reply:
They should update the repo to make it clearer though I agree.

Reply:
There is no mention of MIT in that file. Add a field called "url" in the license object, and link that file there.